### DISCUSSION TITLE
[Feature] Bulk Delete Backed Up Photos

### DISCUSSION THREAD

**Discussion by dannyvfilms** [immich-app/immich#9703] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

This was a core feature of Google Photos or PhotoSync I found valuable. The ability to delete photos from the device that are confirmed to be backed up to Immich. 

Right now my phone storage is full, and I’m pensive to delete photos and videos due to hoping that I don’t accidentally delete something too new that Immich hasn’t backed up yet. 

Being able to look in the backup menu and click “delete backed up files” every now and then would be thoroughly helpful, then I can confirm the deletion in Recently Deleted on my phone’s photos app.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by schuhbacca** [comment]
This will eventually be looked into, but is not a priority until immich is in a stable state. Also duplicate: https://github.com/immich-app/immich/issues/165

**Reply by dannyvfilms** [reply]
Apologies for the duplicate. Thought I had searched well enough.

**Comment by spupuz** [comment]
Congrats in reaching the stable version! I hope this functionality will be developed soon now.
